gearlab-tools.php

```diff
@@ -123,13 +123,12 @@
 add_action('plugins_loaded', function() {
   if (class_exists(Timber::class)) {
     // Timber is running. Extend it!
-    add_filter('get_twig', function(Twig_Environment $twig) {
-      $twig->addFunction(new Twig_SimpleFunction(
-        'gearlab_paginate_links',
-        GearLab\paginate_links::class
-      ));
-
-      return $twig;
+    add_filter('timber/twig/functions', function ($functions) {
```

Comment:
@sdunham I think, you could use `timber/twig` filter instead and make it compatible across the board: https://timber.github.io/docs/v2/hooks/filters/#timber/twig
This would require us to check to make sure proper Twig related classes exist but it should be doable. This would be one way to ensure it will not cause issues with other older projects. Not requirement but suggestion.

Reply:
Great suggestion, I went ahead and implemented it. I still want to make sure any sites still using this plugin specify a version instead of a branch, and work on migrating those sites to the Sitka plugin at some point in the future, but this will provide maximum backward-compatibility. Thanks!